brayns/Brayns.cpp

```diff
@@ -468,62 +479,11 @@ struct Brayns::Impl : public PluginAPI
             loadingProgress += tic;
         }
 
-#if (BRAYNS_USE_BRION)
-        if (!geometryParameters.getCircuitConfiguration().empty())
-            _loadCircuitConfiguration(updateProgress);
-#endif
-
-        if (!geometryParameters.getMolecularSystemConfig().empty())
-            _loadMolecularSystem(updateProgress);
-
         scene.saveToCacheFile();
         scene.buildEnvironmentMap();
         scene.markModified();
     }
 
-#if (BRAYNS_USE_BRION)
-    /**
-        Loads morphologies from circuit configuration (command line
-       parameter --circuit-configuration)
-    */
-    void _loadCircuitConfiguration(const Loader::UpdateCallback& progressUpdate)
```

Comment:
If you remove the progress update (which I guess you do because it doesn't show in the UI anyway), you should mention it in the commit message as well.